meetings/2025-02-26.md

```diff
@@ -28,16 +32,73 @@
+* HZ: I'm in the WG as well. I didn't speak to Ruben for long time, I will make sure his feedback is included.
+
 ### LWS Use Cases
 
+* HZ: There are two repositiories. One is the spec for the protocol; so far not much progress there. There is also use cases repository; many of you already contributed to the issues. We work on a glossary, there is a PR. Now we are working on extractiong requirements from the use cases; they will go into the draft. In the last meeting, Laurence expressed intent to have a protocol draft available in March or early April. 
+* HZ: We are going to close use cases once we have suffcient requirements covering the use case.
+
+### STM Application Launcher
+
+* eP: Given recent work by Michiel and my discussions with ActivityPods team, plus earlier discussions with Joseph Guillaume. I think we could get together for a focused meeting.
+* eP: Mostly an announcement. Michiel recently presented his work. There is also work in ActivityPods. There were quite a few discussions with Joseph. I will create a scheduling poll today and prioritise the times when Joseph could connect and allow him to participate. I will then post it on matrix and the mailing list.
+
+### Blank nodes to be inserted in N3 Patches
+
+See [PR #711](https://github.com/solid/specification/pull/711)
+
+* HZ: There were conversations on matrix and in the PR itself.
+* ML: How can we move forward on this topic?
+* ML: Currently, we can't insert blank nodes. I understand why it is forbidden for deletion. I think for insertions, we should allow them. Ruben seems to be against; I'm not sure if he understood that we just want to allow them in insertions. Sarven approved PR. How should we move forward?
+* eP: I didn't check the minutes of last call. Is Ruben participating? Can Maxime be invited on the LWS call?
+* HZ: LWS WG is not close to that point. I don't recall Ruben in WG calls.
+* HZ: I would propose for me and Maxime to reach out to Ruben and comment and/or approve the PR.
+* JW: Ruben is at Oxford to give a talk, I can talk to him. There seems to be a misunderstanding. There is also a point, should we be proceding with this change given that N3 is not a WG spec. Should we signal possible deprication. Dorthe is discussing making N3 a standard for a while. 
+* HZ: I think the WG is nowhere near making this kind of decisions.
+* eP: useful feedback. What Sarven brought up we discussed on a few occasions. I suggested that in the end we may have to go with SPARQL updates. If the current blank node is a bug in the current spec, should we update the spec with a small update. For Solid, trying to move to SPARQL updates would be a major change, not consistent with the WG request.
+* JW: I agree. A potential phrasing would be that we recommend using SPARQL updates vs N3 patch.
+* eP: sounds reasonable. We could upen a pull request with the desired wording. I had a little chat with PAC about the maturity level. It may be worth running this by PAC - Pierre Antoine.
```

Comment:
```suggestion
* JW: I agree. A potential phrasing would be that we recommend using SPARQL Update rather than N3 Patch.
* eP: Sounds reasonable. We could upen a pull request with the desired wording. I had a little chat with PAC about the maturity level. It may be worth running this by PAC.
```